selfdrive/car/hyundai/values.py

```diff
@@ -27,7 +27,7 @@ def __init__(self, CP):
     self.STEER_STEP = 1  # 100 Hz
 
     if CP.carFingerprint in CANFD_CAR:
-      self.STEER_MAX = 270
+      self.STEER_MAX = 170
```

Comment:
The change in this value will end up applying to many more cars than the 2023 Sportage PHEV, right? Just curious whether the logs and your investigation uncovered a general enhancement?

Reply:
Yes, I only did that to test if it fixed the problems